docs/acr42/01_introduction_and_goals.md

```diff
@@ -0,0 +1,68 @@
+This document follows the [arc42 template](https://arc42.org/).
+# Introduction and Goals
+
+This document describes LAPIS, which is a platform to give easy access to genomic sequence data alongside metadata of
+the sequenced probes. It is used to filter sequence data and return the result to the user through web access, so the
+user can develop his/her own evaluation of the data. The current development is based on two existing
+systems (LAPIS-api and SILO), which are developed for SarsCov2 viruses, and shall be extended to also include other
+virus species.
+
+The following goals have been established for this system:
+
+| Priority |                                                            |
+|----------|------------------------------------------------------------|
+| 1        | Include current functionality of LAPIS in the new approach |
+| 2        | Provide simple setup for new virus species                 |
+| 3        |                                                            |
+
+## Requirements Overview
+
+| Id  | Requirement                |                                                                                                      |
+|-----|----------------------------|------------------------------------------------------------------------------------------------------|
+| F1  | create user instance       | create user instance of the whole system by given configuration for different virus species          |
+| F2  | read in data               | read data provided as aligned and non-aligned sequence data for nucleotides and aminoacids, metadata |
+| F3  | store data                 | store data in compressed form and give fast access                                                   |
```

Comment:
```suggestion
| F3  | store data                 | store data in compressed form                                                  |
```
IMO access is a separate requirement. Compression is important enough as a standalone feature here.